eth-rpc-adapter/src/transports/server-transport.ts

```diff
@@ -20,9 +20,11 @@ export abstract class ServerTransport {
   }
 
   protected async routerHandler({ id, method, params }: JSONRPCRequest, cb?: any): Promise<JSONRPCResponse> {
-    if (id === null || id === undefined || !method) {
+    if (!id || !method) {
```

Comment:
```suggestion
    if (id === null || id === undefined || !method) {
```

Reply:
`!id` will be true if id is `0` or `''`, so usually we still want `id === null || id === undefined` to be more safe